lib/v2/ehentai/ehapi.js

```diff
@@ -86,13 +105,23 @@ async function parsePage(cache, data, get_bittorrent = false, embed_thumb = fals
                     }
                 }
             }
+            if ('ce'.includes(layout)) {
```

Comment:
```suggestion
            if ('le'.includes(layout)) {
```
It should be `le` according to
https://github.com/DIYgod/RSSHub/blob/6d3d8a091b7955c08281a771ce7e68083698e32e/lib/v2/ehentai/ehapi.js#L48-L52